Update ocp-4-19-release-notes.adoc (#98716)
openshift/openshift-docs: release_notes/ocp-4-19-release-notes.adoc

[id="ocp-release-notes-insights-operator-runtime-extractor_{context}"]
==== Insights Runtime Extractor (Technology Preview)

> In this release, the Insights Operator introduces the workload data collection _Insights Runtime Extractor_ feature to help Red{nbsp}Hat better understand the workload of your containers. Available as a Technology Preview, the Insights Runtime Extractor feature gathers runtime workload data and sends it to Red{nbsp}Hat. Red{nbsp}Hat uses the collected runtime workload data to gain insights that can help you make investment decisions that will drive and optimize how you use your {product-title} containers. For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[_Enabling features using feature gates_].

Reviewer: ```suggestion
In this release, the Insights Operator introduces the workload data collection _Insights Runtime Extractor_ feature. Available as a Technology Preview, the Insights Runtime Extractor feature gathers runtime workload data and sends it to Red{nbsp}Hat. Red{nbsp}Hat uses the collected runtime workload data to gain insights that you can then use to make investment decisions that helps improve how you use your {product-title} containers. For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[Enabling features using feature gates].
```
